### DISCUSSION TITLE
V3版本还要等多久才能够兼容replit平台呢？

### DISCUSSION THREAD

**Discussion by Debcharon** [AlistGo/alist#1773] [General]
想要从v2迁至v3版本，但replit平台上执行更新到v3的命令依旧会报错，v3版本还要等多久才能够兼容replit平台呢？

**Comment by ghost** [comment]
把alist-replit 的main.sh 编辑成我这个
# rm -rf alist* data/ #Uncomment this line to update
if [ ! -f "alist" ];then
  curl -L https://github.com/alist-org/alist/releases/download/v3.0.1/alist-linux-musl-amd64.tar.gz -o alist.tar.gz
  tar -zxvf alist.tar.gz
  rm -f alist.tar.gz
  mv alist-linux-musl-amd64 alist
fi
./alist server

**Reply by ghost** [reply]
主要改curl那行和最后的./alist docker 改成server

**Reply by Debcharon** [reply]
感谢，忘了还有server运行的了，之前一直docker一直错
